Form-Controls/styles.css

```diff
@@ -0,0 +1,151 @@
+    color: #1a1a1a; /* Darker text for contrast */
+    padding: 20px 0;
+    text-align: center;
+}
+
+/* Main content */
+main {
+    padding: 20px;
+    background: #ffffff; /* White for main content */
+    margin-top: 20px;
+    border-radius: 5px;
+}
+
+form {
+    display: flex;
+}
+
+.field-wrap {
+    margin-bottom: 40px;
+}
+
+.formbox {
+    display: flex;
+    flex-direction: column;
+    gap: 20px; /* Better than justify-content for consistent spacing */
+    padding: 20px; /* Adds spacing inside the container */
+    border: 1px solid #ccc; /* Optional: adds a border for a defined area */
+    border-radius: 5px; /* Rounded corners for a more modern look */
+    background-color: #f0f0f0; /* Optional: background color for better visibility */
+    width: fit-content;
+}
+
+input, select, button {
+    color: #1a1a1a; /* Darker text for better contrast */
+    background-color: #ffffff; /* White background for form controls */
+    border: 1px solid #999999; /* Darker border for better definition */
+}
+
+button {
+    background-color: #0056b3; /* Darker blue button */
+    color: #ffffff; /* White text for contrast */
+    width: 70%;
+    margin-left: 50px;
+    padding: 10px; /* Adjust padding for better size */
+    height: auto; /* Use auto to ensure the button is responsive */
+    border: none; /* Remove border for cleaner look */
+    border-radius: 5px; /* Rounded corners for button */
+}
+
+.address {
+    display: flex;
+    gap: 10px; /* Creates space between address fields */
+    align-items: center; /* Align items vertically in the middle */
+    flex-wrap: wrap; /* Allows wrapping on smaller screens */
+}
+
+.postcode {
+    flex: 1; /* Flex-grow:1 allows this to take up available space */
+    min-width: 100px; /* Optional: sets a minimum width to maintain layout */
+}
+
+/* Label styling */
+label {
+    transform: translateY(6px);
+    transition: all 0.5s ease;
+    pointer-events: none;
+    font-size: 18px;
+    padding-left: 10px;
+    color: #1a1a1a; /* Darker label text for better contrast */
+}
+
+input, textarea {
+    font-size: 22px;
+    display: block;
+    width: 100%;
+    padding: 5px 10px;
+    background: none;
+    border: 1px solid #888888; /* Darker border for better visibility */
+    color: #1a1a1a; /* Darker text color for better readability */
+    transition: border-color 0.25s ease, box-shadow 0.25s ease;
+}
+
+form {
+    padding: 40px;
+    max-width: 600px;
+    margin: 40px auto;
+    border: 4px solid #f9e3c2; /* Light border */
+    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1); /* Slight shadow for depth */
+}
+
+input:focus, textarea:focus {
+    outline: 0;
+    border-color: #1ab188; /* Highlight border on focus */
+}
+
+/* Footer */
+footer {
+    background: #c4c4c4; /* Darker gray for footer */
+    color: #1a1a1a; /* Dark text for contrast */
+    text-align: center;
+    padding: 10px 0;
+    margin-top: 20px;
+}
+button#choose {
```

Comment:
is there a way we can target the submit button of the form without using its ID?

Reply:
yeah I think there is just one button so we can but I couldn't remeber why I did this ?:)